test_suite/tests/json.rs

```diff
@@ -296,6 +297,7 @@ fn test_recursive_type_with_box() {
         ]
     });
 
+    let registry: PortableRegistry = registry.into();
```

Comment:
I guess you could do `serde_json::to_value::<PortableRegistry>(registry.into())` but that's not better.